spec.emu

```diff
@@ -763,6 +784,55 @@
           1. Let _rest_ be WithClauseToAttributes of |WithEntries|.
           1. Return the list-concatenation of « _entry_ » and _rest_.
         </emu-alg>
+
+        <emu-note>
+          <p><ins>This section is extended by <emu-xref href="#sec-deprecated-assert-keyword-for-import-attributes"></emu-xref>.</ins></p>
+        </emu-note>
+      </emu-clause>
+
+      <emu-clause id="sec-deprecated-assert-keyword-for-import-attributes" deprecated normative-optional>
+        <h1><ins>Deprecated `assert` keyword for Import Attributes</ins></h1>
+
+        <p>Import Attributes are denoted using the `with` keyword. However, there are existing implementations based on a previous version of the proposal using the `assert` keyword. Due to potential web compatibility risks, the `assert` keyword is still included in this specification, but its removal is being investigated.</p>
+
+        <p>Implementations that implement this clause must also implement support for the deprecated `assert` fallback for import attributes in import calls (step <emu-xref href="#step-evaluate-import-call-assert-fallback"></emu-xref> of <emu-xref href="#sec-evaluate-import-call"></emu-xref>).</p>
```

Comment:
This would need to be an iff relationship. But I thought we decided we were going to use a feature test (as in https://github.com/tc39/ecma262/pull/2952), which would accomplish that. Either way, we'll need some solution.